src/patternfly/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,158 @@
+      padding-left: calc(#{$pf-c-dropdown__toggle--PaddingLeft} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      border-width: $pf-c-dropdown__toggle--hover--BorderWidth;
+    }
+
+    .pf-is-expanded & {
+      padding-top: calc(#{$pf-c-dropdown__toggle--PaddingTop} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-right: calc(#{$pf-c-dropdown__toggle--PaddingRight} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-bottom: calc(#{$pf-c-dropdown__toggle--PaddingBottom} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-left: calc(#{$pf-c-dropdown__toggle--PaddingLeft} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      border-color: $pf-c-dropdown__toggle--is-expanded--BorderColor;
+      border-width: $pf-c-dropdown__toggle--is-expanded--BorderWidth;
+    }
+
+    .pf-is-ghost & {
+      background-color: transparent;
+      border-color: transparent;
+    }
+
+    .pf-sans-arrow & {
+      padding-right: $pf-c-dropdown__toggle--PaddingLeft;
+    }
+
+    &::after {
+      box-sizing: content-box;
+      display: inline-block;
+      width: calc(#{$pf-c-dropdown__toggle--FontSize} * .3);
+      height: calc(#{$pf-c-dropdown__toggle--FontSize} * .3);
+      margin-left: $pf-c-dropdown__arrow--MarginLeft;
+      vertical-align: calc(#{$pf-c-dropdown__toggle--FontSize} * .25);
+      content: "";
+      border-style: solid;
+      border-width: $pf-c-dropdown__arrow--BorderWidth $pf-c-dropdown__arrow--BorderWidth 0 0;
+      transform: rotate(135deg);
+
+      .pf-sans-arrow & {
+        content: none;
+      }
+
+      .pf-is-dropup.pf-is-expanded & {
+        vertical-align: calc(#{$pf-c-dropdown__toggle--FontSize} * .1);
+        transform: rotate(-45deg);
+      }
+    }
+  }
+
+  &__menu {
+    position: absolute;
+    z-index: 1000;
+    min-width: 100%;
+    background: $pf-c-dropdown__menu--BackgroundColor;
+    box-shadow: $pf-c-dropdown__menu--BoxShadow;
+
+    .pf-is-alignRight & {
+      right: 0;
+    }
+
+    .pf-is-dropup & {
+      top: 0;
+      transform: translateY(-100%);
+    }
+  }
+
+  &__menu-item {
+    display: block;
+    width: 100%;
+    padding: $pf-c-dropdown__menu-item--PaddingTop $pf-c-dropdown__menu-item--PaddingRight $pf-c-dropdown__menu-item--PaddingBottom $pf-c-dropdown__menu-item--PaddingLeft;
+    color: $pf-c-dropdown__menu-item--Color;
+    text-align: left;
+    white-space: nowrap;
+    border: none;
+
+    &:first-child {
+      padding-top: $pf-c-dropdown__menu-item--first--PaddingTop;
+    }
+
+    &:last-child {
+      padding-bottom: $pf-c-dropdown__menu-item--last--PaddingBottom;
+    }
+
+    &:hover,
```

Comment:
Button has different styles for hover, active, disabled and focus. PF should be consistent.